.github/workflows/changed_file_trigger_examples_check.yml

```diff
@@ -0,0 +1,109 @@
+name: Test Example
+on:
+  pull_request:
+    paths:
+      - 'examples/**'
+    # So only the changes in examples folder will trigger jobs below.
```

Comment:
Usually the comment is put on top of your code, which means it should be above `paths`.

Reply:
OK!